rioting-daisies/BangazonWorkforce
# Jh ticketone2

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.


I have added to the Employee Model since the last push on ticket 1.  There is now a DepartmentName Property as a string so I could have the department name as a join instead of just the Id number.  I created the Index view as well as completed the index portion of the Employee controller.  


Fixes Issue # (link issue ticket here)

## Type of change

Please delete options that are not relevant.


- [ x] New feature (non-breaking change which adds functionality)

# Testing Instructions
git fetch --all
git checkout jh-ticketone2
Test to make sure that you can see the employee first name last name if they are a supervisor or not and that you see the name of the department.  The department name should be HR.





# Checklist:
